docs/electric-api.yaml

```diff
@@ -159,6 +159,23 @@ paths:
 
                 Must be provided as the `shape_id` parameter when making
                 subsequent requests where `offset` is not `-1`.
+            x-electric-schema:
+              schema:
+                type: object
```

Comment:
Question: is this a "correct" type for a header, given that this is always a string? I'm OK with keeping it as an `object`, just curious.